web/src/i18n/locales/en.yaml

```diff
@@ -232,7 +232,7 @@
     statePersonnel: 
       title: "Personnel"
       subheader: "Outline the personnel resources for this section. This includes anyone funded as a part of this activity. This does not include personnel listed in the contract section."
-      helpText: "Personnel costs must include salaries and benefits. FTE is percentage attributable to the activity. Enter individuals only once."
+      helpText: "Personnel costs for this section includes the combined salary and benefits. FTE% is the percentage attributable to the activity. You only need to enter personnel once. Percentages are generated automatically, you only have to insert a number in each section by FFY."
```

Comment:
- Should include some language to indicate that Salary amount is the yearly cost of the individual.
- FTE% -> %FTE to match what's currently being displayed
- "Percentages are generated automatically" is a bit misleading. Maybe "Yearly costs are automatically calculated from the %FTE, provided." ?

Reply:
ok

Reply:
@quarterback What did you think about bullet 3?